backend/src/ml_space_lambda/resource_metadata/lambda_functions.py

```diff
@@ -306,6 +309,40 @@ def _process_endpoint_config_event(details):
             )
 
 
+def _process_emr_event(details):
+    cluster_id = details["clusterId"]
+    cluster_state = details["state"]
+
+    if cluster_state == "TERMINATED":
+        # cluster no longer exists, delete it from the resource table
+        resource_metadata_dao.delete(cluster_id, ResourceType.EMR_CLUSTER)
+    else:
+        cluster_details = emr.describe_cluster(ClusterId=cluster_id)
+        tags = _parse_tags(cluster_details["Cluster"]["Tags"])
+        if "system" in tags and tags["system"] == env_variables["SYSTEM_TAG"]:
+            if "user" in tags and "project" in tags:
+                user = tags["user"]
+                project = tags["project"]
+                metadata = {
+                    "CreationTime": cluster_details["Cluster"]["Status"]["Timeline"]["CreationDateTime"],
```

Comment:
Just want to double check we don't need to call `_convert_timestamp()` on these. I know the APIs aren't all consistent in how they return data so this may be fine.

Reply:
>Creation time
Mar 20, 2024, 16:47:19 EDT

This is what the date looks like in the console right now, so it seems alright